### PR TITLE
[API] Cover zones

### PR DESCRIPTION
**Pull request by Wojdylak**
| Q               | A
|-----------------|-----
| Branch?         | api-platform-3
| Bug fix?        | no
| New feature?    | no
| BC breaks?      | no
| Deprecations?   | no
| Related tickets | N/A
| License         | MIT

### CODE REVIEW

**Review by mpysiak — APPROVED** [review]

File: src/Sylius/Bundle/ApiBundle/spec/Serializer/Denormalizer/ZoneDenormalizerSpec.php [hunk 1]
```diff
@@ -0,0 +1,178 @@
...
+    ): void {
+        $sectionProvider->getSection()->willReturn(new AdminApiSection());
+
+        $this
+            ->supportsDenormalization([], ZoneInterface::class, null, [AbstractNormalizer::OBJECT_TO_POPULATE => $zone])
+            ->shouldReturn(true)
+        ;
+
+        $sectionProvider->getSection()->willReturn(new ShopApiSection());
+        $this
```
Comment (TheMilek) [inline src/Sylius/Bundle/ApiBundle/spec/Serializer/Denormalizer/ZoneDenormalizerSpec.php:48]:
```suggestion
        $sectionProvider->getSection()->willReturn(new ShopApiSection());
        
        $this
```

Comment (TheMilek) [reply src/Sylius/Bundle/ApiBundle/spec/Serializer/Denormalizer/ZoneDenormalizerSpec.php:48]: or both without spacing 🤔

File: tests/Api/Admin/ZonesTest.php [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+                'code' => 'US',
+                'name' => 'UnitedStates',
+                'type' => 'province',
+                'members' => [
+                    ['code' => 'AL'],
+                    ['code' => 'CA'],
+                    ['code' => 'NY'],
+                ],
+            ]),
+        );
```
Comment (TheMilek) [inline tests/Api/Admin/ZonesTest.php]:
```suggestion
        $this->client->request(
            method: 'POST',
            uri: '/api/v2/admin/zones',
            server: $this->headerBuilder()
                ->withJsonLdContentType()
                ->withJsonLdAccept()
                ->withAdminUserAuthorization('api@example.com')
                ->build(),
            content: json_encode([
                'code' => 'US',
                'name' => 'UnitedStates',
                'type' => 'province',
                'members' => [
                    ['code' => 'AL'],
                    ['code' => 'CA'],
                    ['code' => 'NY'],
                ],
            ]),
        );
```

Comment (TheMilek) [reply tests/Api/Admin/ZonesTest.php]: here and in other places 🤔

**Review by TheMilek — APPROVED** [review]

**Pull request merged**